theano/tensor/tests/test_opt.py

```diff
@@ -3034,6 +3037,10 @@ def _verify_stack_trace(self, f):
             self.assertTrue(hasattr(output.variable.tag, 'trace'))
 
     def test_remove_alloc_wo_dimshuffle(self):
+        # Exclude local_useless_alloc, since it does not introduce
+        # assert in all the same cases.
+        self.fast_run_mode = self.fast_run_mode.excluding(
+            'local_useless_alloc')
```

Comment:
I think this will change the value of self for the following tests in that class. I don't think this is a good idea. So don't affect it to self.fast_run_mode, but use it only locally.